src/Libraries/CoreNodes/Math.cs

```diff
@@ -17,7 +17,7 @@ namespace DSCore
     public static class Math
     {
         /// <summary>
-        ///     Generates a random double in the range of [0, 1).
+        ///     Generates a random double in the range of [0, 1].
```

Comment:
Are we sure about this change? I thought it means we would never get 1 as a result but could get 0.

Reply:
You are correct.  Will fix that.